src/standardConfig.js

```diff
@@ -0,0 +1,86 @@
+import importSortPlugin from 'eslint-plugin-simple-import-sort'
+import { createRequire } from 'module'
+
+import { removeProps } from './utils.js'
+
+const require = createRequire(import.meta.url)
+
+export default function standardConfig(opts) {
+  const {
+    // Formatting:
+    prettier,
+    sortImports,
+
+    // Environments:
+    jsx,
+    node,
+    react,
+    typescript,
+    tsconfigRootDir
+  } = opts
+
+  // Check the environment:
+  if (prettier) {
+    try {
+      require('prettier')
+    } catch (_) {
+      throw new Error(
+        'Cannot find prettier. Please run `npm install prettier` or equivalent'
+      )
+    }
+  }
+  if (typescript) {
+    try {
+      require('typescript')
+    } catch (_) {
+      throw new Error(
+        'Cannot find TypeScript. Please run `npm install typescript` or equivalent'
```

Comment:
'Cannot find prettier. Please run `yarn add -D prettier` to install it.'
      )
    }
  }
  if (typescript) {
    try {
      require('typescript')
    } catch (_) {
      throw new Error(
        'Cannot find prettier. Please run `yarn add -D prettier` to install it.'
```suggestion
        'Cannot find prettier. Please run `npm install prettier` or equivalent'
      )
    }
  }
  if (typescript) {
    try {
      require('typescript')
    } catch (_) {
      throw new Error(
        'Cannot find TypeScript. Please run `npm install typescript` or equivalent'
```